joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
joint_controller:
  type: "effort_controllers/JointTrajectoryController"
  joints:
    - BASE_Y
    - BASE_R
    - BASE_P
    - ELBOW_P
    - ELBOW_Y
    - WRIST_P
    - WRIST_Y
    - GRIPPER0
    - GRIPPER1
  gains:
    BASE_Y:
      p: 40.0
      i: 0.0
      d: 4.0
    BASE_R:
      p: 40.0
      i: 0.0
      d: 4.0
    BASE_P:
      p: 40.0
      i: 0.0
      d: 4.0
    ELBOW_P:
      p: 40.0
      i: 0.0
      d: 4.0
    ELBOW_Y:
      p: 40.0
      i: 0.0
      d: 4.0
    WRIST_P:
      p: 40.0
      i: 0.0
      d: 4.0
    WRIST_Y:
      p: 40.0
      i: 0.0
      d: 4.0
    GRIPPER0:
      p: 40.0
      i: 0.0
      d: 4.0
    GRIPPER1:
      p: 40.0
      i: 0.0
      d: 4.0
arm_controller:
  type: "effort_controllers/JointTrajectoryController"
  joints:
    - BASE_Y
    - BASE_R
    - BASE_P
    - ELBOW_P
    - ELBOW_Y
    - WRIST_P
    - WRIST_Y
  gains:
    BASE_Y:
      p: 40.0
      i: 0.0
      d: 4.0
    BASE_R:
      p: 40.0
      i: 0.0
      d: 4.0
    BASE_P:
      p: 40.0
      i: 0.0
      d: 4.0
    ELBOW_P:
      p: 40.0
      i: 0.0
      d: 4.0
    ELBOW_Y:
      p: 40.0
      i: 0.0
      d: 4.0
    WRIST_P:
      p: 40.0
      i: 0.0
      d: 4.0
    WRIST_Y:
      p: 40.0
      i: 0.0
      d: 4.0
gripper_controller:
  type: "effort_controllers/JointTrajectoryController"
  joints:
    - GRIPPER0
    - GRIPPER1
  gains:
    GRIPPER0:
      p: 40.0
      i: 0.0
      d: 4.0
    GRIPPER1:
      p: 40.0
      i: 0.0
      d: 4.0
